src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -392,7 +392,14 @@ class FunctionSegment(BaseSegment):
     type = "function"
     match_grammar = Sequence(
         Sequence(
-            Ref("FunctionNameSegment"),
+            AnyNumberOf(
+                Ref("FunctionNameSegment"),
+                max_times=1,
+                min_times=1,
+                exclude=OneOf(
+                    Ref("ValuesClauseSegment"),
+                ),
+            ),
```

Comment:
Same change as made to ANSI, done for all dialects with their own function segment